WorkflowUI/Sources/Screen/AnyScreen/AnyScreen.swift

```diff
@@ -20,23 +20,18 @@ import UIKit
 
 public struct AnyScreen: Screen {
     /// The original screen, retained for debugging
-    internal let wrappedScreen: Screen
-
-    /// Stored getter for the wrapped screen’s view controller description
-    private let _viewControllerDescription: (ViewEnvironment) -> ViewControllerDescription
+    public let wrappedScreen: Screen
 
     public init<T: Screen>(_ screen: T) {
         if let anyScreen = screen as? AnyScreen {
             self = anyScreen
             return
         }
         self.wrappedScreen = screen
-        self._viewControllerDescription = screen.viewControllerDescription(environment:)
     }
 
     public func viewControllerDescription(environment: ViewEnvironment) -> ViewControllerDescription {
-        // Passed straight through
-        return _viewControllerDescription(environment)
+        return wrappedScreen.viewControllerDescription(environment: environment)
```

Comment:
this is an unrelated cleanup. I couldn't see a reason why this abstraction was necessary. It looks like it's been there since the type was [introduced](https://github.com/square/workflow-swift/commit/fc93feb64e3ad7cdf66e28dc6ee17b566662024d) but at the time the view registry existed so perhaps that had something to do with it

Reply:
hmm yeah also not sure, but seems fine to remove. if you weren't going to store the underlying screen, then i could see why it would exist, so maybe it was here first, and the stored screen value was added after?

Reply:
I thought the same thing but the original implementation stored the screen
```
public struct AnyScreen: Screen {
    internal let wrappedScreen: Screen
    private let viewControllerBuilder: (ViewRegistry) -> AnyScreenViewController.WrappedViewController

    public init<T: Screen>(_ screen: T) {
        self.wrappedScreen = screen
        self.viewControllerBuilder = { viewRegistry in
            return viewRegistry.provideView(for: screen)
        }
    }

    // Internal method to inflate the wrapped screen from a view registry
    func makeViewController(from viewRegistry: ViewRegistry) -> AnyScreenViewController.WrappedViewController {
        return viewControllerBuilder(viewRegistry)
    }
}
```